wger/exercises/tests/test_corrected_exercise.py

```diff
@@ -34,6 +34,7 @@ def correct_exercise(self, fail=True):
                                      'name_original': 'my test exercise',
                                      'license': 2,
                                      'description': description,
+                                     'language': 1,
```

Comment:
@reifred I have tried out your implementation and run the tests. And everything works as desired. I just have one question, what is `language:1`?

Reply:
Thanks @Rhytah for pointing out this. This is because when you check languages.py file we have a function called load_item_languages which returns a list. So for every language loaded I have to call it from a list. So in this case 1 refers to the English language.

Reply:
@reifred Wow, thank you for explaining this.  I learn something new every day. Great work.